jbehave-core/src/test/resources/stream-story.json

```diff
@@ -1 +1 @@
{"keyword": "Outcome:", "value": "ANY", "steps": ["Given a scenario step"]}]},{"keyword": "Scope:", "value": "STORY", "outcomes": [{"keyword": "Outcome:", "value": "ANY", "steps": ["Given an after story step"]}]}]}},"beforeStorySteps": [{"outcome": "successful", "value": "Given a before story step"}],"scenarios": [{"keyword": "Scenario:", "title": "I ask for a loan","givenStories": {"keyword": "GivenStories:", "givenStories":[{"parameters": "", "path": "\/given\/story1"},{"parameters": "", "path": "\/given\/story2"}]},"steps": [{"outcome": "successful", "value": "Given I have a balance of $50"},{"outcome": "ignorable", "value": "!-- Then ignore me"},{"comment": "ignorable", "value": "!-- A comment"},{"outcome": "successful", "value": "When I request $20"},{"outcome": "successful", "value": "When I ask Liz for a loan of $100"},{"outcome": "successful", "value": "When I ask Liz for a loan of $((99))"},{"outcome": "successful", "value": "When I write special chars <>&\""},{"outcome": "successful", "value": "When I write special chars in parameter ((<>&\"))"},{"outcome": "successful", "value": "When I write two parameters ((,,,)) and ((&&&))"},{"outcome": "restarted", "value": "Then I should... - try again", "reason": "hi"},{"story": {"outcome": "restartedStory", "value": "\/path\/to\/story", "reason": "Restarted Story"}},{"cancelled": {"keyword": "STORY CANCELLED", "durationKeyword": "DURATION", "durationInSecs": "2"}},{"outcome": "pending", "keyword": "PENDING", "value": "Then I should have a balance of $30"},{"outcome": "notPerformed", "keyword": "NOT PERFORMED", "value": "Then I should have $20"},{"outcome": "failed", "keyword": "FAILED", "value": "Then I don't return loan", "failure": "org.jbehave.core.model.OutcomesTable$OutcomesFailed"},{"outcomes": {"fields": ["Description","Value","Matcher","Verified"],"values": [["I don't return all","100.0","<50.0>","No"],["A wrong date","01\/01\/2011","\"02\/01\/2011\"","No"]]}}]},{"keyword": "Scenario:", "title": "Parametrised Scenario","examples": {"keyword": "Examples:","steps": ["Given money <money>","Then I give it to <to>"],"parameters": {"names": ["money","to"],"values": [["$30","Mauro"],["$50","Paul"]]}, "examples": [{"keyword": "Example:", "parameters": {"money":"$30","to":"Mauro"},"steps": [{"outcome": "successful", "value": "Given money $30"},{"outcome": "successful", "value": "Then I give it to Mauro"}]},{"keyword": "Example:", "parameters": {"money":"$50","to":"Paul"},"steps": [{"outcome": "successful", "value": "Given money $50"},{"outcome": "successful", "value": "Then I give it to Paul"},{"outcome": "pending", "keyword": "PENDING", "value": "Then I should have a balance of $30"}]}]}}],"afterStorySteps": [{"outcome": "successful", "value": "Given an after story step"}],"pendingMethods": ["@When(\"something \\\"$param\\\"\")\n@Pending\npublic void whenSomething() {\n  \/\/ PENDING\n}\n","@Then(\"something is <param1>\")\n@Pending\npublic void thenSomethingIsParam1() {\n  \/\/ PENDING\n}\n"]}
```

Comment:
could it be formetted?

Reply:
json output generates raw string by default, not formatted

Reply:
It is a test data, what's the actual limitation?

Reply:
it will bring a pain to our fork, but I'm ok to improve it in the master repo

Reply:
test data is compared with data generated by the json output and by default uses string equals, not json nodes comparison

Reply:
So it will be like: "".replaceAll("\\n")
From my point of view, formatting will increase readability and actual changes will be readable

Reply:
what about spaces as well? I would prefer to compare json trees if we are going to refactor this